docs/release_notes.rst

```diff
@@ -0,0 +1,5 @@
+.. release-notes:: Release Notes
+    :collapse-pre-releases:
```

Comment:
I don't see that this option is doing anything:
![image](https://github.com/user-attachments/assets/4cf4ec24-3370-453f-943b-ce5bac462a61)

Reply:
It's not a pre-release, it's a release candidate :shrug: